packages/components/README.md

```diff
@@ -1,35 +1,50 @@
 # PIE Web Components
 
-This readme contains some general information that is relevant for all components. For specific information about a component, please refer to the readme in the component's directory.
+This readme contains some general information that is relevant for all components. For specific information about a
+component, please refer to the readme in the component's directory.
```

Comment:
Were all these newlines automatically added?

Reply:
My IDE re-formatted the file and I didn't noticed until I opened the PR. For now I reverted it so the change is more easily visible.